CHANGELOG.md

```diff
@@ -1,6 +1,220 @@
 Change Log
 ==========
 
-For changes to the Kotlin library, see [kotlin/CHANGELOG.md](kotlin/CHANGELOG.md).
+## Version 0.14.0
+
+_2019-4-16_
+
+# Kotlin
+
+ * Rename WorkflowContext.compose to composeChild. [#274](https://github.com/square/workflow/issues/274)
+ * Rename compose and composeChild to render and renderChild. [#293](https://github.com/square/workflow/issues/293)
+ * Throw if WorkflowContext is accessed after compose returns. [#273](https://github.com/square/workflow/issues/273)
+ * Pass the workflow's scope into initialState. [#286](https://github.com/square/workflow/issues/286)
+ * Add the ability to update the input for a WorkflowHost. [#282](https://github.com/square/workflow/issues/282)
+
+# Swift
+
+ * Kotlin-only release, no code changes.
+
+## Version 0.13.0
+
+_2019-4-12_
+
+# Swift
+
+ * Don't allow `AnyScreen` to wrap itself. ([#264](https://github.com/square/workflow/pull/264))
+
+# Kotlin
+
+ * Upgrade Gradle to 5.x ([#113](https://github.com/square/workflow/issues/113))
+ * Upgrade AGP to 3.3.0 ([#115](https://github.com/square/workflow/issues/115))
+
+## Version 0.12.0
+
+_2019-4-10_
+
+# Swift
+
+ * Improve type safety by removing AnyScreenViewController as a base type. ([#200](https://github.com/square/workflow/pull/200))
```

Comment:
Do we still need to include the issue URLs? Hoping that the pretty rendered release pages will do that for us and we can delete them.

Reply:
We don't need them in the github description. They don't link from regular changelog markdown but that's probably fine, nobody's gonna use the links in this file anyway.